### PR TITLE
Incorporating Kevin's feedback

### PR DESCRIPTION
**Pull request by pazdera**

### CODE REVIEW

File: kwc-dialog.html [hunk 1]
```diff
@@ -70,36 +81,60 @@
...
+                }
+            }
         </style>
         <div class="toolbar">
             <h2>{{title}}</h2>
             <div class="actions">
                 <slot name="actions">
-                    <button class="secondary" dialog-dismiss>Cancel</button>
-                    <button class="primary" dialog-confirm>Done</button>
+                    <button class="secondary" dialog-dismiss>Cancel</button><button class="primary" dialog-confirm>Done</button>
```
Comment (murilopolese) [inline kwc-dialog.html:137]: Was the line break causing issues? It was kind of easier to read, though...

**Review by murilopolese — COMMENTED** [review]
Just a very unimportant comment, other than that double thumbs up.

File: kwc-dialog.html (hunk 1 above)
Comment (pazdera) [reply kwc-dialog.html:137]: The problem there was that having these on separate line inserts a whitespace between the buttons which adds about 5px to the margin. This whitespace doesn't get inserted when you slot some buttons in, weirdly, which then creates an inconsistency.

This is annoying but I couldn't find any other way of getting rid of the space.

Comment (murilopolese) [reply kwc-dialog.html:137]: I feel you pain! heheh All good! It has to do with the `display: inline` thing I thing... I had the same issue with `kwc-soft-buttons`

**Pull request merged**